### DISCUSSION TITLE
Unable to upload files

### DISCUSSION THREAD

**Discussion by jcv99** [requarks/wiki#3954] [Help / Questions] [answered]
I'm not able to upload any file that has more than 0 bytes. I use postgres-11. If it is that i use a wrong database, which one i have to use so I could upload pdf, word, excel and images?

**Comment by NGPixel** [comment, marked as answer]
Do you have a reverse proxy in front of Wiki.js that could block these uploads?

The database is fine and not the cause.

**Reply by jcv99** [reply]
Yep thanks
